data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -6,7 +6,7 @@ set -ex
 IRONIC_IMAGE=$(image_for ironic)
 IRONIC_INSPECTOR_IMAGE=$(image_for ironic-inspector)
 IPA_DOWNLOADER_IMAGE=$(image_for ironic-ipa-downloader)
-COREOS_DOWNLOADER_IMAGE=$(image_for ironic-rhcos-downloader)
+COREOS_DOWNLOADER_IMAGE=$(image_for ironic-machine-os-downloader)
```

Comment:
To ratchet the installer across, this probably has to be:

```sh
COREOS_DOWNLOADER_IMAGE=$(image_for ironic-machine-os-downloader || image_for ironic-rhcos-downloader)
```

or some such.

Reply:
done